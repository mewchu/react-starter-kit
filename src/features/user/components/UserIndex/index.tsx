import React, { SFC } from 'react';
import Box from '@material-ui/core/Box';
import UserList from '../UserList/container';
import { UserSearch } from '../UserSearch';
import { UserCreate } from '../UserCreate';

const UserIndex: SFC = () => {
  return (
    <>
      <Box mb="10px" display="flex" justifyContent="space-between">
        <UserSearch />
        <UserCreate />
      </Box>
      <UserList />
    </>
  );
};
export default UserIndex;
