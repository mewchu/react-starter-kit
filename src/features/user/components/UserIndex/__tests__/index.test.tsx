import React from 'react';
import { shallow } from 'enzyme';
import UserIndex from '..';

test('<UserIndex /> snapshot', () => {
  const wrapper = shallow(<UserIndex />);
  wrapper.debug();
  expect(wrapper).toMatchSnapshot();
});
