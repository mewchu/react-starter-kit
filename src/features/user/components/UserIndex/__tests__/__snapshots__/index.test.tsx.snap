// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UserIndex /> snapshot 1`] = `
<Fragment>
  <Styled(MuiBox)
    display="flex"
    justifyContent="space-between"
    mb="10px"
  >
    <UserSearch />
    <UserCreate />
  </Styled(MuiBox)>
  <Wrapper />
</Fragment>
`;
